Scripts/Source/UD_RandomRestraintManager.psc

```diff
@@ -275,7 +285,7 @@ bool Function LockAnyRandomRestrain(Actor akActor,int iNumber = 1,bool bForce =
     while iNumber
         iNumber -= 1
         loc_res = loc_res || LockRandomRestrain(akActor)
-		Utility.wait(3)	 ; small delay to ensure of no equips running into one another
+        Utility.wait(1)     ; small delay to ensure of no equips running into one another
```

Comment:
This should be not needed, as lock function is mutexed. Adding this will make locking the devices slower with not additional safety added.